var/spack/repos/builtin/packages/spatial-index/package.py

```diff
@@ -7,9 +7,9 @@
 class SpatialIndex(PythonPackage):
     """Spatial indexer for geometries and morphologies"""
 
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/hpc/SpatialIndex"
-    git      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
-    url      = "ssh://bbpcode.epfl.ch/hpc/SpatialIndex"
+    homepage = "https://bbpgitlab.epfl.ch/hpc/SpatialIndex"
+    git      = "ssh://git@bbpgitlab.epfl.ch:2201/hpc/SpatialIndex.git"
```

Comment:
2201 for external use?

Reply:
Now the default SSH port is used, both inside and outside EPFL network \o/